pkg/snapstore/snapstore_suite_test.go

```diff
@@ -41,7 +38,6 @@ var _ = BeforeSuite(func() {
 	logrus.Infof("Starting test server...")
 	th.SetupHTTP()
 	initializeMockSwiftServer(testObj)
-	swiftStore = snapstore.NewSwiftSnapstoreFromClient(bucket, prefix, "/tmp", 5, fake.ServiceClient())
```

Comment:
Any reason for removing this?

Reply:
I didn't see how is it helping. Aren't we setting up mock swift store before every unit test case?